src/ts/edit.ts

```diff
@@ -8,19 +8,23 @@
 declare let key: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 import { notif, reloadElement } from './misc';
 import { getTinymceBaseConfig, quickSave } from './tinymce';
-import { EntityType, Target, Upload, Payload, Method, Action, PartialEntity } from './interfaces';
+import { EntityType, Target, Upload, Model, Payload, Method, Action, PartialEntity } from './interfaces';
 import './doodle';
 import tinymce from 'tinymce/tinymce';
 import { getEditor } from './Editor.class';
 import { getEntity } from './misc';
 import Dropzone from 'dropzone';
-import { File } from 'dropzone';
+import type { DropzoneFile } from 'dropzone';
 import i18next from 'i18next';
 import { Metadata } from './Metadata.class';
 import { Ajax } from './Ajax.class';
 import UploadClass from './Upload.class';
 import EntityClass from './Entity.class';
 
+class CustomDropzone extends Dropzone {
```

Comment:
it's great that you managed to deal with dropzone types, I remember trying to solve it without success.

Reply:
Took me some time as well.